impeller/geometry/point.h

```diff
@@ -204,6 +204,10 @@ struct TPoint {
     return *this - axis * this->Dot(axis) * 2;
   }
 
+  constexpr Radians AngleTo(const TPoint& p) const {
+    return Radians{std::atan2(this->Cross(p), this->Dot(p))};
```

Comment:
FWIW, I was looking at Chromium code related to this recently and there's some note about atan2 not being precise enough on macOS. I can try to dig it up if we think it's important here.

We may want to document that this method is expensive and backed by atan2, although probably anyone who cares enough to read it will know that...

Reply:
I'd be interested in seeing this -- I wonder if the libcxx we use is the same and if it leaves us in a different situation. High precision happens to not be super important for how we're using it in the blur, but it may become important for future uses.